src/meta/model_v2/migration/src/m20240630_131430_remove_parallel_unit.rs

```diff
@@ -0,0 +1,110 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .alter_table(
+                Table::alter()
+                    .table(WorkerProperty::Table)
+                    .add_column(
+                        ColumnDef::new(WorkerProperty::Parallelism)
+                            .integer()
+                            .not_null(),
+                    )
+                    .to_owned(),
+            )
+            .await?;
+
+        manager
+            .alter_table(
+                Table::alter()
+                    .table(WorkerProperty::Table)
+                    .drop_column(WorkerProperty::ParallelUnitIds)
+                    .to_owned(),
+            )
+            .await?;
```

Comment:
I saw you talked about compatibility, but it seems the specific behavior here is not clearly discussed: We persisted parallel unit ids for tables, but dropped columns and added new columns here, so what will happen exactly for existing jobs?

Reply:
> We won't use the parallel unit id of actors and the vnode mapping field of fragments. All the necessary information can be derived from existing data.

I'm still a little confused. e.g., IIUC we can have a table scheduled to a given worker with a fixed parallelism.

Now we dropped the persisted info (the table's parallel units), how can we recover it?

BTW, I think I commented in the wrong place. I should have commented on `Fragments`

Reply:
Thanks, I didn't notice `ActorStatus` is part of `TableFragments`.

> The ParallelUnit in ActorStatus has not been removed because I haven't figured out how to handle backward compatibility elegantly yet. 🥵 We will only use the WorkerId field. If it's a new ActorStatus, we will fill ParallelUnit with u32::MAX as the id.

BTW, this might be worth writing in the proto comments.